ted/src/main/java/frc/robot/commands/FeederLaunchNote.java

```diff
@@ -0,0 +1,91 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo - 2024
+// File: FeedNoteCommand.java
+// Intent: Forms a command to feed the note to the shooter or dunker. 
```

Comment:
change comment to reflect what this command actually does. This is a command to shoot a note assuming the shooter is already at the right angle and speed and does not require the shooter outfeed subsystem.